src/Rating/Rating.mdx

```diff
@@ -0,0 +1,63 @@
+---
+name: Rating
+route: /extras/rating
+menu: Extras
+---
+
+import { Playground, PropsTable } from 'docz';
+import Rating from '../Rating';
+import { RatingStar } from '../Rating';
+
+# Rating
+
+## Import
+
+`import { Rating } from 'fannypack'`
+
+## Basic Usage
+
+The rating callback returns `{ rating, maxRating }`.
+
+<Playground>
+  <Rating  onRate={({ rating }) => console.log(rating)}/>
+</Playground>
+
+## Controlled
```

Comment:
This title is misleading. To me, "controlled" means that the state will be controlled by the user, and not `Rating` itself. Maybe **Default values** is more appropriate.

Reply:
It's not exactly controlled in the traditional sense. I'll update the docs to be more descriptive.